vayesta/ewf/uewf.py

```diff
@@ -224,3 +224,62 @@ def get_atomic_ssz(self, dm1=None, dm2=None, atoms=None, projection='sao', dm2_w
                         pb = proj_x[ix][b]
                         ssz[a,b] += (np.sum(tmpa*pb[0]) + np.sum(tmpb*pb[1]))/4
         return ssz
+
+    def _get_dm_corr_energy_old(self, global_dm1=True, global_dm2=False):
+        """Calculate correlation energy from reduced density-matrices.
+
+        UHF ONLY!
+
+        Parameters
+        ----------
+        global_dm1 : bool
+            Use 1DM calculated from global amplitutes if True, otherwise use in cluster approximation. Default: True
+        global_dm2 : bool
+            Use 2DM calculated from global amplitutes if True, otherwise use in cluster approximation. Default: False
+
+        Returns
+        -------
+        e_corr : float
```

Comment:
Maybe the sphinx doesn't like this entry without a description

Reply:
Strange that it only fails for python 3.8.

Reply:
The docs are only built on the 3.8 job in the CI